src/Pyz/Zed/Book/BookDependencyProvider.php

```diff
@@ -0,0 +1,46 @@
+<?php
+namespace Pyz\Zed\Book;
+
+use Pyz\Zed\Book\Business\BookService;
+use Spryker\Zed\Kernel\AbstractBundleDependencyProvider;
+use Spryker\Zed\Kernel\Container;
+
+/**
+ * @method \Pyz\Zed\Book\BookConfig getConfig()
+ */
+class BookDependencyProvider extends AbstractBundleDependencyProvider
+{
+    /**
+     * @var string
+     */
+    public const BOOK_SERVICE = 'BOOK_SERVICE';
+
+    /**
+     * @param \Spryker\Zed\Kernel\Container $container
+     *
+     * @return \Spryker\Zed\Kernel\Container
+     */
+    public function provideBusinessLayerDependencies(Container $container): Container
+    {
+        $container->set(static::BOOK_SERVICE, function (Container $container) {
+            return new BookService();
```

Comment:
we use DependencyProvider to build dependency on other modules.
Inside  the module you should use Factory